packages/stress/src/mode/chat/root_chat.ts

```diff
@@ -127,6 +130,20 @@ export async function startStressChat(opts: {
     for (let i = 0; i < clients.length; i += PARALLEL_UPDATES) {
         const span = clients.slice(i, i + PARALLEL_UPDATES)
         const results = await Promise.allSettled(span.map((client) => joinChat(client, chatConfig)))
+        if (chatConfig.kickoffMessageEventId && chatConfig.countClientsMessageEventId) {
+            const reactionCount = await countReactions(
+                clients[0],
+                chatConfig.announceChannelId,
+                chatConfig.kickoffMessageEventId,
+            )
+            await updateCountClients(
+                clients[0],
+                chatConfig.announceChannelId,
+                chatConfig.countClientsMessageEventId,
+                chatConfig.clientsCount,
+                reactionCount,
+            )
+        }
```

Comment:
this only runs on process one after waiting for each span of process 0's clients. Its not going to report any clients that join after process 0 completes.
 
we should kick off an async task before this block that counts up the reactions in a loop with a reasonable sleep timeout and then cancel it when we stop the clients. Make sense? We can call it reportStats and add more stats to it as we get more sophisticated.

Reply:
Approved in case you want to check it in and see things working, but we should iterate on it.